src/test/game/BoardTest.java

```diff
@@ -2,148 +2,222 @@
 
-        assertEquals(emptyBoard, board.availableMoves());
+        assertEquals(9, board.availableMoves().size());
     }
 
     @Test
     public void returnsAListOfOneMove() {
-        Board board = new Board(asList("1", "O", "X", "O", "X", "O", "X", "O", "X"));
-        List<String> possibleMoves = asList("1");
+        Board board = new Board(3, asList(EMPTY, O, X, O, X, O, X, O, X));
+        List<Integer> possibleMoves = asList(0);
 
-        assertEquals(possibleMoves, board.availableMoves());
+        assertThat(board.availableMoves(), is(possibleMoves));
     }
 
     @Test
     public void playersCantEnterSameMoveWhenMoveIsX() {
-        Board board = new Board(asList("X", "2", "3", "4", "5", "6", "7", "8", "9"));
+        Board board = new Board(3, asList(X, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY));
 
         assertFalse(board.isMoveAvailable(1));
     }
 
     @Test
     public void playersCantEnterSameMoveWhenMoveIsO() {
-        Board board = new Board(asList("O", "2", "3", "4", "5", "6", "7", "8", "9"));
+        Board board = new Board(3, asList(O, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY));
 
         assertFalse(board.isMoveAvailable(1));
     }
 
     @Test
     public void boardHasNoAvailableMovesLeft() {
-        Board board = new Board(asList("X", "O", "X", "O", "X", "O", "X", "O", "X"));
+        Board board = new Board(3, asList(X, O, X, O, X, O, X, O, X));
 
         assertFalse(board.hasAvailableMoves());
     }
 
     @Test
     public void canScoreAHorizontalWin() {
-        Board board = new Board(asList("X", "X", "X", "4", "5", "O", "O", "8", "9"));
-
-        assertTrue(board.findWin(PLAYER_ONE));
+        Board board = new Board(3, asList(
+                X, X, X,
+                EMPTY, EMPTY, EMPTY,
+                EMPTY, EMPTY, EMPTY));
+        Board board2 = new Board(3, asList(
+                EMPTY, EMPTY, EMPTY,
+                X, X, X,
+                EMPTY, EMPTY, EMPTY));
+        Board board3 = new Board(3, asList(
+                EMPTY, EMPTY, EMPTY,
+                EMPTY, EMPTY, EMPTY,
+                X, X, X));
+
+        assertTrue(board.findWin(X));
+        assertTrue(board2.findWin(X));
+        assertTrue(board3.findWin(X));
+    }
+
+    @Test
+    public void canScoreWinForAColumnOnAThreeByThree() {
+        Board board = new Board(3, asList(
+                X, EMPTY, EMPTY,
+                X, EMPTY, EMPTY,
+                X, EMPTY, EMPTY));
+        Board board2 = new Board(3, asList(
+                EMPTY, X, EMPTY,
+                EMPTY, X, EMPTY,
+                EMPTY, X, EMPTY));
+        Board board3 = new Board(3, asList(
+                EMPTY, EMPTY, X,
+                EMPTY, EMPTY, X,
+                EMPTY, EMPTY, X));
+
+        assertTrue(board.findWin(X));
+        assertTrue(board2.findWin(X));
+        assertTrue(board3.findWin(X));
+    }
+
+    @Test
+    public void canScoreWinForAColumnOnAFourByFour() {
```

Comment:
I don't think you need to test every possible column win... I'm just thinking - what if your mentors ask you to create a 5x5 and a 12x12 board as well? Or even to allow the board size to be specified by the user? I think just one example of a column win board is enough for the test.